stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -717,8 +717,8 @@ end
 
 Compute the rank of a matrix by counting how many singular
 values of `A` have magnitude greater than `max(atol, rtol*σ₁)` where `σ₁` is
-`A`'s largest singular values, atol and rtol are the absolute and relative
-tolerance respectively.
+`A`'s largest singular value, `atol` and `rtol` are the absolute and relative
+tolerance, respectively.
```

Comment:
tolerances

Reply:
Also put a period after "singular value"  (or an "and", but that is awkward with the "and" already between atol and rtol).